platforms/axplat-aarch64-phytium-pi/src/init.rs

```diff
@@ -0,0 +1,63 @@
+use axplat::init::InitIf;
+
+#[allow(unused_imports)]
+use crate::config::devices::{GICC_PADDR, GICD_PADDR, TIMER_IRQ, UART_IRQ, UART_PADDR};
+use crate::{config::plat::PSCI_METHOD, mem::phys_to_virt};
+
+fn hart_to_logid(hard_id: usize) -> usize {
+    crate::config::devices::CPU_ID_LIST
+        .iter()
+        .position(|&x| x == hard_id)
+        .unwrap()
+}
+
+struct InitIfImpl;
+
+#[impl_plat_interface]
+impl InitIf for InitIfImpl {
+    /// Initializes the platform at the early stage for the primary core.
+    ///
+    /// This function should be called immediately after the kernel has booted,
+    /// and performed earliest platform configuration and initialization (e.g.,
+    /// early console, clocking).
+    fn init_early(cpu_id: usize, _dtb: usize) {
+        axcpu::init::init_cpu(hart_to_logid(cpu_id));
```

Comment:
`cpu_id` is already logic ID here, we should convert hard ID to logic ID before `axplat::call_main`.